pkg/interface/src/views/landscape/components/ResourceSkeleton.tsx

```diff
@@ -172,7 +208,7 @@ export function ResourceSkeleton(props: ResourceSkeletonProps): ReactElement {
           flexShrink={0}
           ref={actionsRef}
         >
-          {canWrite && <WriterControls />}
+          {ExtraControls()}
```

Comment:
+1 for no closures as components

Reply:
Thank you for reading the React source code to uncover this wisdom